transports/transport-highway/src/main/java/org/apache/servicecomb/transport/highway/HighwayConfig.java

```diff
@@ -18,13 +18,26 @@
 package org.apache.servicecomb.transport.highway;
 
 import com.netflix.config.DynamicIntProperty;
+import com.netflix.config.DynamicLongProperty;
 import com.netflix.config.DynamicPropertyFactory;
 import com.netflix.config.DynamicStringProperty;
 
 public final class HighwayConfig {
   private HighwayConfig() {
   }
 
+  public static final String KEY_SERVICECOMB_REQUEST_WAIT_IN_POOL_TIMEOUT = "servicecomb.highway.server.requestWaitInPoolTimeout";
```

Comment:
maybe no need to 2 different config item for this?

Reply:
2 item is more flexible. I test it for highway this configuration works much better than rest.